src/server/config/database.ts

```diff
@@ -43,7 +43,8 @@ class MongoConnection {
         await mongoose.connect(mongoUrl, opts);
       } else {
         console.log('Connecting to MongoDB...');
-        const uri = process.env.ATLAS_URI || 'mongodb+srv://NO:CONNECTION@STRING.FOUND.mongodb.net/FAILURE?retryWrites=true&w=majority';
+        // const uri = process.env.ATLAS_URI || 'mongodb+srv://NO:CONNECTION@STRING.FOUND.mongodb.net/FAILURE?retryWrites=true&w=majority';
+        const uri = 'mongodb+srv://hack4impact:boilerplate@cluster0.4jd7b.mongodb.net/Boilerplate?retryWrites=true&w=majority'
```

Comment:
please avoid uploading sensitive content such as the MongoDB uri to github. One reason this might not be working is because 'dotenv/config' is not imported into this file. I have added it so it should be working now

Reply:
sweet thanks, resolved!